src/base-element.js

```diff
@@ -482,6 +482,22 @@ export class BaseElement {
   }
 
   /**
+   * Utility method that bubbles events from the given element
+   * to this element.
+   * @param  {!Array<string>} events
```

Comment:
Can we make this take a `string|!Array<string>`? It'll make single events cleaner to bubble.